crates/ibc/src/core/ics03_connection/handler.rs

```diff
@@ -1,4 +1,22 @@
-//! This module implements the processing logic for ICS3 (connection open handshake) messages.
+//! This module implements the processing logic for ICS3 (connection open
```

Comment:
I would really like for us to employ this naming convention in all the core modules (in the future). 
So should we generalize this documentation and move it to the `crates/ibc/src/core/mod.rs`?

Reply:
[f6ce907](https://github.com/cosmos/ibc-rs/pull/170/commits/f6ce907d741d1ca0945bfa9e66ced8d943c7e934)

Reply:
Actually what I had in mind was to move these doc comments out of here (i.e. delete them from here), but I also don't mind if we choose to keep them here.

Reply:
Ugh, that was my intention as well! [9c9b8a9](https://github.com/cosmos/ibc-rs/pull/170/commits/9c9b8a93b4f90ce86a8780fbf35c7fb115563c3c)